config/wpcalypso.json

```diff
@@ -71,6 +71,7 @@
 		"jetpack/features-section/atomic": true,
 		"jetpack/features-section/jetpack": true,
 		"jetpack/features-section/simple": true,
+		"jetpack/magic-link-signup": true,
```

Comment:
This feature flag enables the magic link signup, and it was already enabled on all environment (production, stage, development, horizon), except wpcalypso, which is the one on Calypso Live Branches.